src/WebJobs.Script.WebHost/Diagnostics/KubernetesEventGenerator.cs

```diff
@@ -20,34 +22,62 @@ public KubernetesEventGenerator(IEnvironment environment, Action<string> writeEv
 
-            _writeEvent($"{(int)ToEventLevel(level)},{subscriptionId},{appName},{functionName},{eventName},{source},{NormalizeString(details)},{NormalizeString(summary)},{hostVersion},{formattedEventTimeStamp},{exceptionType},{NormalizeString(exceptionMessage)},{functionInvocationId},{hostInstanceId},{activityId},{runtimeSiteName},{slotName},{_podName}");
+            JObject traceLog = new JObject();
+            traceLog.Add("Level", (int)ToEventLevel(level));
+            traceLog.Add("SubscriptionId", subscriptionId);
+            traceLog.Add("AppName", appName);
+            traceLog.Add("FunctionName", functionName);
+            traceLog.Add("EventName", eventName);
+            traceLog.Add("Source", source);
+            traceLog.Add("Details", NormalizeString(details));
+            traceLog.Add("Summary", NormalizeString(summary));
+            traceLog.Add("HostVersion", ScriptHost.Version);
+            traceLog.Add("EventTimeStamp", eventTimestamp.ToString(EventTimestampFormat));
+            traceLog.Add("ExceptionType", exceptionType);
+            traceLog.Add("ExceptionMessage", NormalizeString(exceptionMessage));
+            traceLog.Add("FunctionInvocationId", functionInvocationId);
+            traceLog.Add("HostInstanceId", hostInstanceId);
+            traceLog.Add("ActivityId", activityId);
+            traceLog.Add("RuntimeSiteName", runtimeSiteName);
+            traceLog.Add("SlotName", slotName);
+            traceLog.Add("PodName", _podName);
+
+            _writeEvent(traceLog.ToString(Formatting.None));
         }
 
         public override void LogFunctionMetricEvent(string subscriptionId, string appName, string functionName, string eventName, long average, long minimum, long maximum, long count, DateTime eventTimestamp, string data, string runtimeSiteName, string slotName)
         {
-            string hostVersion = ScriptHost.Version;
+            JObject metricEvent = new JObject();
+            metricEvent.Add("SubscriptionId", subscriptionId);
+            metricEvent.Add("AppName", appName);
+            metricEvent.Add("FunctionName", functionName);
+            metricEvent.Add("EventName", eventName);
+            metricEvent.Add("Average", average);
+            metricEvent.Add("Minimum", minimum);
+            metricEvent.Add("Maximum", maximum);
+            metricEvent.Add("Count", count);
+            metricEvent.Add("HostVersion", ScriptHost.Version);
+            metricEvent.Add("EventTimeStamp", eventTimestamp.ToString(EventTimestampFormat));
+            metricEvent.Add("Data", NormalizeString(data));
+            metricEvent.Add("RuntimeSiteName", runtimeSiteName);
+            metricEvent.Add("SlotName", slotName);
 
-            _writeEvent($"{subscriptionId},{appName},{functionName},{eventName},{average},{minimum},{maximum},{count},{hostVersion},{eventTimestamp.ToString(EventTimestampFormat)},{NormalizeString(data)},{runtimeSiteName},{slotName},{_podName}");
```

Comment:
why are these being deleted?

Reply:
It is a no-op today.

Reply:
Can you please expand on that? I see these methods are using `_writeEvent` as well - This PR is explicitly removing logs. Am I missing something here?

Reply:
LogFunctionsDetailsEvent is not for Kusto logging today. Today we only use LogFunctionTraceEvent and LogFunctionsMetricEvent for Kusto logging. So we only need to send those events to the shared log analytics.